core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -780,6 +780,15 @@ public String toString() {
               + "is set to 0, the cache will be disabled, and "
               + "`alluxio.user.file.metadata.load.type=Once` will behave like `Always`.")
           .build();
+  public static final PropertyKey MASTER_UFS_BLOCK_LOCATION_CACHE_CAPACITY =
```

Comment:
can you put this property in alphabetical order. e.g., before `MASTER_UFS_PATH_CACHE_CAPACITY`

Reply:
done